src/pages/GameLobby.tsx

```diff
@@ -31,10 +31,7 @@ const GameLobby: React.FunctionComponent = () => {
 
   return (
     <Container>
-      <Header>
-        <h2>Game Lobby</h2>
-        <ButtonToHome />

```

Comment:
does this mean there is no button to home i can't tell yet but wanted to comment now